python/cudf/bindings/orc.pyx

```diff
@@ -57,14 +57,15 @@ cpdef cpp_read_orc(path, columns=None, stripe=None, skip_rows=None,
     orc_reader.source = source_ptr
 
     usecols = columns
+    arr_cols = []
+    orc_reader.use_cols_len = 0
     cdef vector[const char*] vector_use_cols
     if usecols is not None:
-        arr_cols = []
         for col in usecols:
             arr_cols.append(str(col).encode())
-        vector_use_cols = arr_cols
-        orc_reader.use_cols = vector_use_cols.data()
         orc_reader.use_cols_len = len(usecols)
+    vector_use_cols = arr_cols
```

Comment:
Using an if-else block with the other path be `orc_reader.use_cols = NULL` and `orc_reader.use_cols_len = 0` makes more sense and matches the other sections. But this is fine.